### DISCUSSION TITLE
Usage with twin.macro

### DISCUSSION THREAD

**Discussion by MrBlenny** [vechai/vechaiui#36] [Q&A] [answered]
Amazing stuff @pepelele, I'm super excited to give this a go for my next project.

I'm new to a bunch of this next-gen tailwind / css in js tooling but was hoping to use verchai ui along with [twin.macro](https://github.com/ben-rogerson/twin.macro) / emotion. I gave this a go (using [this template](https://github.com/ben-rogerson/twin.examples/tree/master/vite-emotion-typescript)) but it seems that the verchai styles were not loading. Do you have any advise on the compatibility with this type of tooling?

**Comment by pepelele** [comment, marked as answer]
Issue https://github.com/ben-rogerson/twin.macro/issues/432 ... The `twin.macro` doesn't support `apply` syntax ... I'm having the same problem with WindiCSS ... maybe I must to rewrite using css-in-js syntax instead of `apply` syntax
